_past_events/5-2018-rome.md

```diff
@@ -0,0 +1,20 @@
+---
+city: Rome
+year: 2018
+
+link: http://rome.rustfest.eu
+
+sponsors:
+  - 1aim
+  - Cryptape
+  - Ferrous Systems
+  - Mozilla
+  - Parity
+  - Baidu X-Lab
+  - Kodebox
+  - Threema
+  - Mullvad
+  - openSUSE
+  - rrbone
+  - PingCap
+---
```

Comment:
Maybe add a link to the videos? Like:
```
videos: https://media.ccc.de/b/conferences/rustfest/2018-Rome
```

Reply:
The videos attribute is not used anywhere yet (and we will publicly announce the video later this week)